core/includes/module.inc

```diff
@@ -379,15 +379,21 @@ function module_load_all_includes($type, $name = NULL) {
  * - Invoke hook_modules_installed().
  * - Invoke hook_modules_enabled().
  *
- * @param string[] $module_list
+ * @param string $module_list
  *   An array of module names.
```

Comment:
Is it a string or an array?

Reply:
Ha, good point!

Reply:
We don't strictly enforce this sort of type checking, but `string[]` means an array of strings, as is recognized by VSCode and PHPStorm.